primap2/csg/_strategies/null.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import xarray as xr
+
+import primap2
+
+
+class NullStrategy:
+    """return a nan time-series (used to skip certain fixed coordinate combinations)"""
```

Comment:
I think this one is obsolete with `exclude_*`, no?

Reply:
I think so. I forgot to remove it before the pr.